shuffle/cli/src/main.rs

```diff
@@ -145,21 +156,19 @@ fn normalized_address(account_address: Option<String>) -> Result<AccountAddress>
                 input_address
             }
         }
-        None => get_latest_address()?,
+        None => get_latest_address(home)?,
     };
     Ok(AccountAddress::from_hex_literal(
         normalized_string.as_str(),
     )?)
 }
 
-fn get_latest_address() -> Result<String> {
-    let home = shared::Home::new(shared::get_home_path().as_path())?;
+fn get_latest_address(home: &Home) -> Result<String> {
     home.check_account_path_exists()?;
     Ok("0x".to_owned() + &fs::read_to_string(home.get_latest_address_path())?)
```

Comment:
Let's improve the validation of the on disk value by using `Address` or `AccountAddress`:

```rust
AccountAddress.from_hex(fs::read_to_string(home.get_latest_address_path())).to_hex_literal();
```